File: terre14.py
import sys

def liste_est_triee(liste):
    for i in range(len(liste) - 1):
        if liste[i] > liste[i + 1]:
            return False
    return True

ma_liste = sys.argv[1:]

try:
    ma_liste = [int(x) for x in ma_liste]      
    if liste_est_triee(ma_liste):
        print("La liste est triée.")
    else:
        print("La liste n'est pas triée.")
except ValueError:
    print("erreur.")

File: terre14.js
const args = process.argv.slice(2)

if (args.length < 2) {
  console.error("Le programme a besoin d'au moins 2 arguments.")
  process.exit()
}

const numbers = []

for (const arg of args) {
  if (isNaN(arg)) {
    console.error("Le programme a besoin d'une liste de nombres entiers.")
    process.exit()
  }
  numbers.push(+arg)
}

let isSorted = true

for (let i = 1; i < numbers.length; i++)
  if (numbers[i] < numbers[i - 1]) isSorted = false

if (isSorted) {
  console.log("Triée !")
} else console.log("Pas Triée !")
